homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -26,37 +29,152 @@
 SENSOR_TYPE_HUMIDITY = "humidity"
 SENSOR_TYPE_BATTERY = "battery"
 SENSOR_TYPE_CO2 = "CO2"
+SENSOR_TYPE_POWER = "power"
+SENSOR_TYPE_VOLTAGE = "voltage"
+SENSOR_TYPE_CURRENT = "electricCurrent"
 
-METER_PLUS_SENSOR_DESCRIPTIONS = (
-    SensorEntityDescription(
-        key=SENSOR_TYPE_TEMPERATURE,
-        device_class=SensorDeviceClass.TEMPERATURE,
-        state_class=SensorStateClass.MEASUREMENT,
-        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+SENSOR_DESCRIPTIONS_BY_DEVICE_TYPES = {
```

Comment:
Since a lot of entity descriptions are the same for every device, can we maybe create them beforehand and then just reference them here?

An example can be found in LG Thinq
https://github.com/home-assistant/core/blob/5f36062ef339bc77a2fdb8997f4d2ae0bb198228/homeassistant/components/lg_thinq/sensor.py#L244-L271

Reply:
I predefine all the entity descriptions and reference them here. Could you please review the updated code at your earliest convenience?